DeAceroDLabs/marketplace-frontend
# [DMP-716] Display Offer Detail

## What is done

[[DMP-716] Desplegar detalle de oferta seleccionada](https://deacero.atlassian.net/browse/DMP-716)

## How is done

1. Created new components to display the offer detail.
2. In OfferDetail (new component) was added a useEffect to get the detail of each article.
3. Modified some types to match the cartItems types to the offer articles types.

## How to test

1. Run the project via `npm start`
2. Add some products to your shoppingcart, I'm including an image with the ones that I used for getting the three different offers.
3. Go to carrito/ofertas
4. Wait for the offers to show up.
5. Click on each option and see how the products of each offer are displayed.

<img width="748" alt="Screenshot 2022-12-20 at 12 57 23" src="https://user-images.githubusercontent.com/68872548/209750628-c6cf5499-8c9f-4a82-8137-d84ab4f3e7d3.png">
